pkg/agent/metrics/prometheus.go

```diff
@@ -94,6 +90,18 @@ func InitializePrometheusMetrics() {
 	// and will not measure anything unless the collector is first registered.
 	gaugeHost.Set(1)
 
+	InitializePodMetrics()
+	initializeNetworkPolicyMetrics()
```

Comment:
it's a bit surprising that 2 of these functions are exported and one isn't

Reply:
I left it for a future change when network policy metrics are tested in a similar way.

Reply:
I think we should expert it for consistency even if we don't test it yet